scripts/check-golint.sh

```diff
@@ -18,6 +18,14 @@ set -e
 set -u
 set -o pipefail
 
+# Set a GOBIN to install golint.
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE}")/.." && pwd -P)"
```

Comment:
Code inspired from https://github.com/kubernetes/kubernetes/blob/master/hack/update-bazel.sh#L30 as suggested by @BenTheElder and @@ixdy